fulp_modules/wintercoats/code/wintercoats.dm

```diff
@@ -0,0 +1,31 @@
+// This is a code file for everything to do with wintercoats made by fulp. - Homingpenguins
+
+/obj/item/clothing/suit/hooded/wintercoat/fulp //the baseline fulp wintercoat, so i dont have to repeat the same exact lines every 5 seconds.
```

Comment:
the baseline fulp wintercoat, so i dont have to repeat the same exact lines every 5 seconds.

What do you mean by this? You're only adding one wintercoat